site/app/templates/grading/electronic/ta_status/StatusStaticAnalysis.twig

```diff
@@ -0,0 +1,149 @@
+                mode: 'markers',
+                text: text2,
+                textposition: 'auto',
+                marker: {
+                    color: getComputedStyle(document.documentElement).getPropertyValue('--standard-light-yellow-brown'),
+                    opacity: 1,
+                    line: {
+                        color: 'rgb(170,170,170)',
+                        width: 0.5
+                    }
+                }
+            }
+
+            var trace3 = {
+                x: x3,
+                y: y3,
+                name: '0-50% or lower (for penalty)',
+                type: 'scatter',
+                mode: 'markers',
+                text: text3,
+                textposition: 'auto',
+                marker: {
+                    color: getComputedStyle(document.documentElement).getPropertyValue('--error-alert-dark-red'),
+                    opacity: 1,
+                    line: {
+                        color: 'rgb(170,170,170)',
+                        width: 0.5
+                    }
+                }
+            }
+
+            var data = [trace1, trace2, trace3];
+
+            var layout = {
+                title: `${xLabel} vs. Autograding Scores`,
+                barmode: 'overlay',
+                xaxis: {
+                    title: xLabel
+                },
+                yaxis:{
+                    title: 'Autograding Scores'
+                },
+                font: {
+                    family: '"Source Sans Pro", sans-serif',
+                }
+            };
+
+            Plotly.newPlot('myDiv4', data, layout, { displayModeBar: false, displaylogo: false });
+        }
+
+        document.getElementById('x-axis-selector').addEventListener('change', function() {
+            var selectedValue = this.value;
+            if (selectedValue === 'runtime') {
+                updatePlot(runtime, 'Total Runtime');
+            } else if (selectedValue === 'memory') {
+                updatePlot(memory, 'Memory Usage');
+            } else if (selectedValue === 'sloc') {
+                updatePlot(sloc, 'Source Lines of Code');
+            }
+        });
+
+        // Initial plot
+        updatePlot(runtime, 'Total Runtime');
+    </script>
+    <div id="mySmallDiv4">
+        <script>
+            var sum = runtime.reduce((a, b) => a + b, 0);
+            var average = Math.floor(sum / runtime.length);
+            var max = Math.max(...runtime);
+            var min = Math.min(...runtime);
+            document.write(`<b>Total Runtime: </b>` + "<br>" + `Maximum: ` + `${max}` + '&nbsp&nbsp&nbsp&nbsp' +
+                `Minimum: ` + `${min}` + '&nbsp&nbsp&nbsp&nbsp' + 'Average: ' + `${average}`);
+            document.write("<br>" + "<br>");
+            document.write(`<b>Autograding: </b>` + "<br>" + `Maximum possible: ` + {{ autograding_non_extra_credit }} +
+                '&nbsp&nbsp&nbsp&nbsp' + 'Average: ' + {{ autograded_average.getAverageScore() }});
```

Comment:
Directly injecting values into HTML like this could lead to an XSS attack.  It would be better to hide/show the HTML and change the text dynamically via JS, while keeping the underlying structure the same.